src/api/environment.cc

```diff
@@ -407,6 +407,7 @@ MaybeLocal<Object> GetPerContextExports(Local<Context> context) {
   Local<Object> exports = Object::New(isolate);
   if (context->Global()->SetPrivate(context, key, exports).IsNothing())
     return MaybeLocal<Object>();
+  InitializePrimordials(context);
```

Comment:
This is only hit for vm contexts and main contexts created without snapshot support